tasks/Module 3/task_1.py

```diff
@@ -0,0 +1,59 @@
+"""Task 1 of Module 3."""
+import pyinputplus as pyip
+
+BREAD = {'wheat': 1.321, 'white': 2.5, 'sourdough': 40}
+PROTEIN = {'chicken': 32.3, 'turkey': 55, 'ham': 1, 'tofu': 0.1}
+CHEESE = {'cheddar': 5, 'Swiss': 5.5, 'mozzarella': 1000}
+SAUCE = {'mayo': 300, 'mustard': 400, 'lettuce': 500, 'tomato': 600}
+
+sandwich_ingredients = []
+
+
+def sandwich_maker():
+    """Provide information on ordered sandwich/es.
+
+    Based on users input, provides information on:
+    number of ordered sandwiches;
+    sandwich's ingredients;
+    order's price
+    """
+    while True:
```

Comment:
What is the purpose of this loop? It passes only one time anyway because of the `break` statement at the end so please remove it.

Reply:
Loop removed